bot/bot.py

```diff
@@ -52,11 +56,27 @@ async def setup_hook(self) -> None:
 
     async def on_error(self, event: str, *args, **kwargs) -> None:
         """Log errors raised in event listeners rather than printing them to stderr."""
+        e_type, e_val, e_tb = exc_info()
```

Comment:
3.11 has [`sys.exception`](https://docs.python.org/3/library/sys.html#sys.exception), which is nicer to use.

Reply:
f9054b0

Reply:
f9054b0